### PR TITLE
prometheus-rabbitmq-exporter: 0.25.2 -> 1.0.0-RC7.1

### PR DESCRIPTION
**Pull request by cole-h**
Otherwise, the root certificate from cloudAMQP is unsupported.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/monitoring/prometheus/rabbitmq-exporter.nix [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+  version = "1.0.0-RC7.1";
 
   goPackagePath = "github.com/kbudde/rabbitmq_exporter";
 
   src = fetchFromGitHub {
     owner = "kbudde";
     repo = "rabbitmq_exporter";
-    rev = "8238c5d9ad509dab94e631cdecf3d75155fbdee4";
-    sha256 = "1zb21jms2wlmiw5vhk1wyhv0xrv5a9y08y9q6x92hv727kjqjr6w";
+    rev = "v1.0.0-RC7.1";
```
Comment (SuperSandro2000) [inline pkgs/servers/monitoring/prometheus/rabbitmq-exporter.nix:12]:
```suggestion
    rev = "v{version}";
```

File: pkgs/servers/monitoring/prometheus/rabbitmq-exporter.nix [hunk 2]
```diff
@@ -1,17 +1,16 @@
-# This file was generated by https://github.com/kamilchm/go2nix v2.0-dev
 { lib, stdenv, buildGoPackage, fetchFromGitHub }:
 
 buildGoPackage {
```
Comment (SuperSandro2000) [inline pkgs/servers/monitoring/prometheus/rabbitmq-exporter.nix:3]:
```suggestion
buildGoPackage rec {
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Can we use buildGoModule here?

**Comment by SuperSandro2000** [comment]
> Can we use buildGoModule here?

I still have that question. Maybe a thing todo for another PR.

**Comment by cole-h** [comment]
I don't know the difference between the two, so I'd leave it for another PR. Feel free to handle that.

**Pull request merged**